docs/user-guide/configuration.md

```diff
@@ -0,0 +1,60 @@
+###...
+The docker volume plugin at the moment supports vSphere and Photon platforms. All the volume provisioning and management operations are supported on both the platforms. Both the platforms can be used to provision VMDK based volumes and can be attached to containers. 
+
+The configurations used by a driver while performing operations are read from a JSON file and the default location where it looks for it /etc/docker-volume-vsphere.conf. You can also override it to use  a different configuration file by providing --config option and the full path to the file. Finally the parameters passed on the CLI override the one from the configuration file.
+
+## vSphere Docker volume Driver
+
+The vSphere volume driver can be used on a standalone or cluster of ESX servers via the ESX service. ESX service needs to be installed and running on all servers. The volumes are created on ESX host using the VIM (Virtual Infrastructure Management) APIs.
+
+The configuration for vSphere docker volume driver require the driver type which can be one of vsphere or vmdk (For backwards compatibility). You can also provide the log related configurations.
+
+```
+{
+    "Driver": "vsphere"
+    "MaxLogAgeDays": 28,
+    "MaxLogSizeMb": 100,
+    "LogPath": "/var/log/docker-volume-vsphere.log",
```

Comment:
sorry L17 sounds confusing. It belongs to vSphere docker volume plugin. 

can you please revisit the flow of this document and correct as per need?